containers/proxy-systemd-services/uyuni-proxy-services.config

```diff
@@ -2,11 +2,23 @@
 # the EnvironmentFile services property is pointing there
 
 # Where to get the images from.
-# It should contain the registry FQDN and path to the proxy-* images without trailing slash 
-NAMESPACE=registry.opensuse.org/uyuni
+# It should contain the registry FQDN and path to the proxy-* images without trailing slash
+# Define individual namespaces for each service or let them use the default namespace
+# HTTPD_NAMESPACE=
+# SALT_NAMESPACE=
+# SQUID_NAMESPACE=
+# SSH_NAMESPACE=
+# TFTP_NAMESPACE=
+DEFAULT_NAMESPACE=registry.opensuse.org/uyuni
 
 # Tag of the images to pull
-TAG=latest
+# Define individual tags for each service or let them use the default tag
+# HTTPD_TAG=
+# SALT_TAG=
+# SQUID_TAG=
+# SSH_TAG=
+# TFTP_TAG=
+DEFAULT_TAG=latest
```

Comment:
Thank you. That's some very valid question. Technically the systemd services are a package, so a user could install some other version we provide and restart it. 
@cbosdo whaT do you think?

Reply:
ok makes sense

Reply:
@cbosdo @aaannz We could have [a CLI tool like this one](https://github.com/uyuni-project/uyuni/pull/6240/files#diff-a43f41ab3451e56e4d4b3a195f208ff3a0e4d23d21ff9f02952e45bfcd04cd3d) to control the systemd case in general. For now it supports two commands:
* update
    * config in the form of `proxy update config -s httpd -t 0.1.0 -r registry.opensuse.org/uyuni`. It would create a separate cfg file for a service and restart it. 
* reset `proxy.sh reset -s httpd` removes the separate cfg file and restarts the service.

Every service has a reference to the file even if it doesn't exist by default. There's no error in this case.

The template is more or less safe to use and extensible, so we could use it for more complex cases I guess. In case of a PTF we could give a customer simply a command to execute.